metric-imperial-converter/sample.env

```diff
@@ -1,2 +1,3 @@
 PORT=
+DB=
```

Comment:
I think you meant to add the `DB` environment variable to https://github.com/freeCodeCamp/demo-projects/pull/34 instead of this PR.
```suggestion
```